skopt/callbacks.py

```diff
@@ -222,6 +222,30 @@ def _criterion(self, result):
             return None
 
 
+class RepeatedMinStopper(EarlyStopper):
+    """Stop the optimization when there is no improvement in the minimum.
+
+    Stop the optimization when there is no improvement in the minimum
+    achieved function evaluation after `n_best` iterations.
+    """
+    def __init__(self, n_best=50):
+        super(EarlyStopper, self).__init__()
+        self.n_best = n_best
+        self.count = 0
+        self.minimum = np.finfo(np.float).max
+
+    def _criterion(self, result):
+        if result.fun < self.minimum:
+            self.minimum = result.fun
+            self.count = 0
+        elif result.fun > self.minimum:
+            self.count = 0
+        else:
+            self.count += 1
```

Comment:
Is there a typo in how the if statements are written? It seems like the `else` clause would never get evaluated and so we never trigger the stop?? 😕